### DISCUSSION TITLE
Need help regarding setup Robotics academy

### DISCUSSION THREAD

**Discussion by NAF1S** [JdeRobot/RoboticsAcademy#1912] [Q&A] [answered]
**Hello everyone**, I am Lokman sharif. I want to contribute to the Robotics academy. What I have done by now that fork and clone this repo on my ubuntu system. Now I need some guidelines on how I can access the full system. I have installed Django, react, and docker. Can anyone give me a complete guideline to go through the academy?

Thanks in advance.

**Comment by jmplaza** [comment]
Please read the [documentation](https://github.com/JdeRobot/RoboticsAcademy/tree/master/docs). We are in the process of improving that documentation and you always have the source code. There is also a nice [introductory video](https://www.youtube.com/watch?v=Y6CoOK9WtDQ) at the JdeRobot's YouTube channel.

**Comment by NAF1S** [comment, marked as answer]
hello @jmplaza ,thanks for your reply.After reading the documentation I tried to setup the academy and I getting this error.Can you help to get rid of this error.
![Screenshot from 2023-02-01 23-18-18](https://user-images.githubusercontent.com/111578019/216117751-1e4efaf3-1ee1-44ca-a57d-45466cedd688.png)

**Comment by NAF1S** [comment]
Hello @jmplaza , I fix that error myself. Now I can host the academy. Can you tell me what to do further?
